src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -176,6 +183,11 @@ export default {
 
     #oc-breadcrumb-contextmenu-trigger > span {
       vertical-align: middle;
+      border: 3px solid transparent;
+      &:hover {
+        background-color: var(--oc-color-background-hover);
```

Comment:
There is no `--oc-color-background-hover` in the design system. Would you like to add it (and respectively also prepare a color for the dark mode) or use one of the existing background colors in http://localhost:6060/#/Design%20Tokens ?

Reply:
already is defined :)